sipa/templates/register/identify.html

```diff
@@ -0,0 +1,28 @@
+{% extends "register/base.html" %}#
+
+{% macro skip_btn() %}
+{% if skip_verification %}
+    <button type="submit" class="btn btn-danger" name="skip_verification">{{ _("Verifizierung überspringen") }}</button>
+{% endif %}
+{% endmacro %}
+
+{% block register_content %}
+    {{ forms.render_simple(form=form, form_id="register-form", submit_text=_("Weiter"), extra_btn=skip_btn) }}
```

Comment:
Situation-aware: If there was a mistake, label it `Continue`

Reply:
Also, the abort button could also be labeled „skip _automatic_ verification“ to make it more clear that it does not abort his application, it just delays the verification.

Reply:
CC @georgeto